tests/pages/pay-page.js

```diff
@@ -1,16 +1,34 @@
-import { clickable, collection, create, text, triggerable, visitable } from 'ember-cli-page-object';
+import { clickable, collection, create, text, visitable } from 'ember-cli-page-object';
 import AccountDropdown from 'codecrafters-frontend/tests/pages/components/account-dropdown';
 import Header from 'codecrafters-frontend/tests/pages/components/header';
 
 export default create({
   accountDropdown: AccountDropdown,
-  clickOnApplyRegionalDiscountButton: clickable('[data-test-apply-regional-discount-button]'),
+
+  clickOnChoosePlanButton: clickable('[data-test-choose-plan-button]'),
```

Comment:
All of this (choose plan, extra invoice details, proceed to checkout etc.) should be nested inside chooseMembershipPlanModal, not placed directly on the pay-page object